refactor: 🚚 rename obj in package properties script to `package_properties`

signekb opened the pull request:
# Description

When I renamed the template, I forgot to rename the `PackageProperties` object in it to match. This PR does that. 

This PR needs a quick review.

## Checklist

- [X] Added or updated tests
- [X] Ran `just run-all`

signekb committed:
refactor: :truck: rename obj in package properties script to `package_properties`

signekb committed:
test: :white_check_mark: update tests with properties obj name `package_properties`

martonvago reviewed (dismissed):
Does this need to be updated in the docs as well?

signekb commented:
Yes! On it :)

lwjohnst86 committed:
Merge branch 'main' into refactor/rename-properties-obj-in-package-properties-script

signekb committed:
docs: :memo: update guides with new obj and script name

`properties.py` -> `package_properties.py`; `properties` -> `package_properties` within code blocks.
